docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -166,6 +166,15 @@ implementation is used:
     `query_partition_filter_required` catalog session property for temporary,
     catalog specific use.
   - `false`
```

Comment:
Please shorten commit message to
`Add query-partition-filter-required-schemas config to iceberg`

Reply:
Thank you for the review! I updated the commit message according to your guidance.